CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ Full documentation for hipSOLVER is available at [hipsolver.readthedocs.io](http
 ### Deprecated
 ### Removed
 ### Fixed
+- Fixed potential workspace allocation failure in some functions.
```

Comment:
This is too vague. Please give enough detail so that someone experiencing this failure can tell that it's been fixed.